schedule_lessons/dashboard/static/dashboard/css/agenda.css

```diff
@@ -281,15 +283,48 @@ html body {
   color: white;
 }
 
+#desktopNav{
+  display: block !important;
+}
+
+#navbarSupportedContent{
+  display: none;
+}
+
+@media(max-width: 991.98px) {
+  #myDropdown {
+    margin-right: -110px !important;
+  }
+
+  #notificationsDropdownPane {
+    height: 350px;
+    width: 300px;
+    margin-right: -240px !important;
+  }
+
+  #noti-triangle-up {
+    margin-right: 265px;
+  }
+
+  #pro-triangle-up {
+    margin-right: 120px;
+  }
+  
+  #desktopNav{
+    display: none !important;
+  }
+
+  #searchBar { 
+    left: 0%;
+  }
+}
+
 @media(min-width: 576px) {
+
```

Comment:
nit: remove newline

Reply:
About the code quality: We need to do some major work on the code in all the css files. There are unwanted, duplicate and unnecessary classes, things are sort of hard to find and overall a little messy. I'll take a thorough look at the code and we'll open a new PR if necessary.

Reply:
sounds great man! would really appreciate that